proto/internalapi/central/compliance_operator.proto

```diff
@@ -224,3 +224,23 @@ message ComplianceOperatorScanStatusV2 {
     google.protobuf.Timestamp  start_time        = 7;
     google.protobuf.Timestamp  end_time          = 8;
 }
+
+message ComplianceOperatorScanSettingBindingV2 {
+      string                                  id                  = 1;
+      string                                  name                = 2;
+      string                                  cluster_id          = 3;
```

Comment:
I have not been putting the cluster_id in these internal api objects as that field comes from the pipeline, not sensor itself.  I recommend removing it.